### ISSUE TITLE
Text slider

### ISSUE THREAD

**Issue by george-kar** [uikit/uikit#3104]
I need to write a text slider but it looks like the beta version doesn't have a component. Slideshow is very limited to image and it adds a hardcoded height that cannot be easily controlled ie to create dynamic height based on text size. It looks like yootheme gave priority on components really need for their themes (PRO version) and left the rest (like slideset, date/time component) out.

Anyone made it already and can provide a working example?

**Comment by aarongerig** [comment]
As far as I know, the next beta version will include a slider component, which would fit your needs. So stay tuned for that!

**Comment by eladnova** [comment]
Really looking forward to this

**Comment by saschadube** [comment]
The new [Slideshow](https://getuikit.com/docs/slideshow) component is now an image slideshow only. This decision was made because of the animation effects between slides. The fade, scale, pull and push animations all require a completely opaque and covering area (like an image) otherwise they would look good. Also we don't need to take care of the height because all slides should have the same height.

The new Slider component mentioned by @aarongerig will be the same like [Slider in UIkit 2](https://getuikit.com/v2/docs/slider.html).

What you are looking for is our [Switcher](https://getuikit.com/docs/switcher) component. It allows to switch between content tabs.

Does this help you?

**Comment by george-kar** [comment]
@saschadube I disagree. If I completely remove images and hardcode fix the height, text sliding works good with animation (not all though). It can be a text slider too.

Unfortunately, switcher doesn't cover my needs as slideshow allows you to have previous next buttons, etc

**Comment by viocassel** [comment]
I agree with @george-kar. 
@saschadube, Switcher `previous`/ `next` switch **only** the adjacent items... 😞

**Comment by saschadube** [comment]
@george-kar @viocassel Support for a previous and next buttons is already there. See [documentation](https://getuikit.com/docs/switcher#navigation-controls)

```
<a href="" uk-switcher-item="next"></a>
<a href="" uk-switcher-item="previous"></a>
```
I noticed the markup may not work the same like for the slideshow. We have to test it. But it can easily be fixed.

Please let us know all requirement for the text slider.

**Comment by aarongerig** [comment]
@viocassel @george-kar Here's a simple example of how you could achieve something alike with some of the currently available UIkit components: https://codepen.io/aarongerig/pen/KZXVjy

**Comment by george-kar** [comment]
@aarongerig  that was a great example :) thanks
